inference-engine/src/mkldnn_plugin/nodes/roifeatureextractor_onnx.cpp

```diff
@@ -156,11 +157,12 @@ void ROIAlignForward_cpu_kernel(
       offset_bottom_rois++;
     }
 
+    T offset = aligned ? (T)0.5 : (T)0.0;
```

Comment:
@taka-no-me @gladilov-gleb , should we align this change with the VPU side?

Reply:
New attribute doesn't break layer in VPU for old cases (otherwise we will see failings in private pre-commit). So changes looks ok to me.

@andrejsokolov FYI